### PR TITLE
bugzilla: copy all comments when cloning bug

### PR DESCRIPTION
**Pull request by AlexNPavel**
This changes the way that the description of the bugzilla clone is formatted to be more in line with what bugzilla's web UI clone does.

/cc @alvaroaleman

### CODE REVIEW

File: prow/bugzilla/client.go [hunk 1]
```diff
@@ -268,18 +268,32 @@ func cloneBugStruct(bug *Bug, subcomponents map[string][]string, comments []Comm
...
-		newBug.CommentIsPrivate = desc.IsPrivate
-		newBug.CommentTags = desc.Tags
-		newBug.IsMarkdown = desc.IsMarkdown
-	} else {
-		// This cases _shouldn't_ happen. But just in case...
-		newBug.Description = fmt.Sprintf("This is a clone of Bug #%d.", bug.ID)
-		// We don't know whether this bug should be private. Default to private in case this bug contains private information.
-		newBug.CommentIsPrivate = true
+	for _, comment := range comments {
+		if comment.IsPrivate {
```
Comment (alvaroaleman) [inline prow/bugzilla/client.go:272]: So if the bug we are cloning contains a single private comment we make the cloned bugs description private? Is that what the UI cloning does as well or does it just omit the private comments?

Comment (AlexNPavel) [reply prow/bugzilla/client.go:272]: The UI cloning just makes the entire description private if any comments are private.

Comment (AlexNPavel) [reply prow/bugzilla/client.go:272]: I would've preferred to put all public comments in a public description and then maybe make a second private comment with all the private comments from the parent afterwards. But that could be a bit confusing, and for now we'll just stick with what the UI does.

**Pull request merged**